data.sql

```diff
@@ -5,3 +5,13 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    ('Gabumon', '2018-11-15', 2, true, 8),
    ('Pikachu', '2021-01-07', 1, false, 15.04),
    ('Devimon', '2017-05-12', 5, true, 11);
+
+-- update table
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander','2020-02-08', 0, false,  11),
+('Plantmon','2021-11-15', 2, true,  5.7),
+('Squirtle','1993-04-02', 3, false,  12.13),
+('Angemon','2005-06-12', 1, true,  45),
```

Comment:
* Kindly add the right weight values for the highlighted queries as according to the requirements are negative numbers 
![image](https://user-images.githubusercontent.com/52497246/235605799-dcc2ae18-48fb-4459-8cf8-095c2a8d72cb.png)